Azure/azure-cli
# [AppService] Fixes #16041: az webapp config ssl create results in unknown error

**Description**
Fixes #16041 

`az webapp config ssl create` results in an unknown error. This is caused by python SDK not set to handle 202 responses meant for polling

- Will make update to SDK to handle 202 response, but in the mean time we can catch when SDK throws error due to 202 response and poll on the certificate to see when its created

**Testing Guide**
- `az webapp config hostname add --resource-group {{resource-group-name}} --webapp-name {{app-service-name}} --hostname {{hostname}}`
- `az webapp config ssl create --hostname {{hostname}} --name {{app-service-name}} --resource-group {{resource-group-name}}`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
